cmd/kots/cli/enable-ha.go

```diff
@@ -17,6 +17,7 @@ func EnableHACmd() *cobra.Command {
 		Use:           "enable-ha",
 		Short:         "Enables HA mode for the admin console",
 		Long:          ``,
+		Deprecated:    "This command is deprecated and will be removed in a future release.",
```

Comment:
Message printed would be:

```
$ ./bin/kots enable-ha --help   
Command "enable-ha" is deprecated, This command is deprecated and will be removed in a future release.
```

So I propose:

```suggestion
		Deprecated:    "and will be removed in a future release.",
```